### DISCUSSION TITLE
[Bug] update to1.3.1 and upgrade db,but full node size still 86g.

### DISCUSSION THREAD

**Discussion by changpeng75** [Chia-Network/chia-blockchain#11048] [Support]
### What happened?

I delete the node file and re-synced. But the db file still have 86g.
Run :
chia db validate
Return :
FAILED: Database is missing version table

### Version

1.3.1

### What platform are you using?

Linux

### What ui mode are you using?

CLI

**Comment by shaneo257** [comment]
did you do the steps here https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference#upgrade
